### PR TITLE
Request to merge

### PR DESCRIPTION
**Pull request by Koushith**
This Branch includes :
 - Home, About, Team, and Contact pages.
 - Added React Router
 - Replaced placeholder image with actual illustrations. 
 - Tablet Responsiveness.

Upcoming Changes : 
- [ ] Color schemes, mobile responsiveness,  and making pixel perfect according to the mockup.

### CODE REVIEW

**Comment by abhinav-anshul** [comment]
-  Replace `class` with `className` in all the files as we are using `JSX` 
-  Maintain consistent naming convention for files, In the About folder, There is AboutUs.js as well as About_description, Therefore I would suggest either use `camel case` or `snake case`.

File: tailblocks/src/App.js [hunk 1]
```diff
@@ -1,22 +1,34 @@
...
+          <Route exact path='/'>
+            <Home />
+          </Route>
+          <Route path='/about'>
+            <About />
+          </Route>
+          <Route path='/team'>
+            <Team />
+          </Route>
+          <Route path='/contact'>
```
Comment (koshikraj) [inline tailblocks/src/App.js:26]: I think we should add an error page too. Right now it just shows an empty component.

File: tailblocks/src/Pages/About.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Header from '../components/Header';
+import Description from '../components/Description';
+import AboutDescription from '../components/About/About_description';
```
Comment (koshikraj) [inline tailblocks/src/Pages/About.js]: Let's use a consistent naming convention (`PascalCase` for the component files)

File: tailblocks/src/components/Teams/Members.js [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+        </div>
+        <div class='flex flex-wrap -m-4'>
+          <div class='p-4 lg:w-1/4 md:w-1/2'>
+            <div class='h-full flex flex-col items-center text-center'>
+              <img
+                alt='team'
+                class='flex-shrink-0 rounded-lg w-full h-56 object-cover object-center mb-4'
+                src='https://dummyimage.com/203x203'
+              />
+              <div class='w-full'>
```
Comment (koshikraj) [inline tailblocks/src/components/Teams/Members.js]: Shall we maintain the team details in a separate config file and import and iterate it here just to reduce the file size.

**Comment by koshikraj** [comment]
Overall a pretty great PR as there are a lot of updates here :+1:. Good PR description too. Make sure to name the branch and commit messages according to our naming conventions in the future.

File: tailblocks/src/App.js (hunk 1 above)
Comment (Koushith) [reply tailblocks/src/App.js:26]: Done. added 404 error page.

File: tailblocks/src/components/Teams/Members.js (hunk 3 above)
Comment (Koushith) [reply tailblocks/src/components/Teams/Members.js]: created a new config file that contains an array of objects and mapped in the Team section.

**Comment by Koushith** [comment]
Changes include  : 
- Added 404 error page.
- Maintained the team details in a separate config file and imported it on the Team page and mapped accordingly. 
- corrected the naming conventions.
- Fixed the bottom Navigation section.

**Review by abhinav-anshul — APPROVED** [review]
LGTM, Merging

File: tailblocks/src/App.js [hunk 4]
```diff
@@ -1,21 +1,37 @@
-import React from 'react'
-import Header from './components/Header'
-import Hero from './components/Hero'
-import Description from './components/Description'
-import Solution from './components/Solution'
-import Technology from './components/Technology'
-import Footer from "./components/Footer"
+import React from 'react';
+import Home from './pages/home';
```
Comment (koshikraj) [inline tailblocks/src/App.js]: Looks like this path doesn't exist anymore. Check the case in each import. It causes issues on Mac and Linux.

File: tailblocks/src/components/About/AboutUs.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import About from '../../pages/about';
```
Comment (koshikraj) [inline tailblocks/src/components/About/AboutUs.js]: Looks like this path doesn't exist anymore. Check the case in each import. It causes issues on Mac and Linux.

File: tailblocks/src/App.js (hunk 4 above)
Comment (Koushith) [reply tailblocks/src/App.js]: Removed all unnecessary imports.

File: tailblocks/src/components/About/AboutUs.js (hunk 5 above)
Comment (Koushith) [reply tailblocks/src/components/About/AboutUs.js]: Removed all unnecessary imports.

**Pull request merged**